src/frontend/scripts/src/generateIcon.ts

```diff
@@ -0,0 +1,37 @@
+import fs from "fs";
+import path from "path";
+import {
+  exportTemplate,
+  iconTemplate,
+  indexTemplate,
+  lineToFindInStories,
+  storyTemplate,
+} from "./generateIcon.templates";
+import {pathToComponents} from "./componentPath";
+import writeFile from "./tools/writeFile";
+import appendToFile from "./tools/appendToFile";
+import createDirectory from "./tools/createDirectory";
+
+const iconName = process.argv[2];
+if (!iconName) throw new Error("Invalid icon name provided");
+
+const fullPath = path.join(pathToComponents, `/svgIcons/${iconName}`);
+
+createDirectory(fullPath);
+
+// INDEX.TS
+writeFile(path.join(fullPath, "index.ts"), indexTemplate(iconName));
+
+// ICONNAME.TSX
+writeFile(path.join(fullPath, `${iconName}.tsx`), iconTemplate(iconName));
+
+// EXPORT IN ICONS' INDEX.TS
+appendToFile(path.join(fullPath, "../index.ts"), exportTemplate(iconName));
+
+// ADD ICON IN STORY
+appendToFile(
+  path.join(fullPath, "../SVGIcon.stories.tsx"),
```

Comment:
This file seems to be missing, did I forgot something to do?

Reply:
Ok, I see, thanks for your clarification.
Maybe you could edit this PR to target [sidebar-frontend-components](https://github.com/openfun/jitsi-magnify/tree/sidebar-frontend-components) instead of [main](https://github.com/openfun/jitsi-magnify) then ?